posts/2020-12-22-ocp-log-forwarding.adoc

```diff
@@ -325,7 +340,8 @@ spec:
 
 Alternatively, you can modify this file on the web console. Ensure you are in the `openshift-logging` project namespace and go to `Administration > Custom Resource Definitions > ClusterLogging > Instances > Instance > YAML`. Change the `managementState` field from "Managed" to "Unmanaged".
 
-Next, in the Fluentd config map, change the merge_json_log attribute of the `kubernetes.var.log.containers.**` filter to true. If you are running on OpenShift 4.3 to 4.5, this step is slightly different. Skip to the line that says *OpenShift 4.3 to 4.5 only*.
+[start=2]
+. In the Fluentd config map, change the `merge_json_log` attribute of the `kubernetes.var.log.containers.` filter to `true`. If you are running on OpenShift 4.3 to 4.5, this step is slightly different. Skip to the line that says *OpenShift 4.3 to 4.5 only*.
```

Comment:
Ah I see that turned everything bold after. We can just italicize instead?

Reply:
Realized I could fix it with a line break!